scipy/integrate/_quadrature.py

```diff
@@ -627,26 +651,111 @@ def simpson(y, x=None, dx=1.0, axis=-1, even='avg'):
         if x.shape[axis] != N:
             raise ValueError("If given, length of x along axis must be the "
                              "same as y.")
+
+    # even keyword parameter is deprecated
+    if 'even' in kwds:
+        warnings.warn(
+            "The 'even' keyword will be removed in a future release of scipy",
+            DeprecationWarning
+        )
```

Comment:
```suggestion
        warnings.warn(
            "The 'even' keyword is deprecated as of SciPy 1.11.0 and will be "
            "removed in SciPy 1.13.0",
            DeprecationWarning, stacklevel=2
        )
```
Also can you add a test that this appears as expected